static/gsApp/views/amCheckout/cart.tsx

```diff
@@ -238,13 +239,16 @@ function ItemsSummary({activePlan, formData}: ItemsSummaryProps) {
                               budgetTerm:
                                 activePlan.budgetTerm === 'pay-as-you-go'
                                   ? 'PAYG'
-                                  : activePlan.budgetTerm,
+                                  : displayBudgetName(activePlan, {title: true}),
                             })
                           ) : (
                             // "Unlock with on-demand" gets cut off in non-chonk theme
                             <Text size="xs">
                               {tct('Unlock with [budgetTerm]', {
-                                budgetTerm: activePlan.budgetTerm,
+                                budgetTerm:
+                                  activePlan.budgetTerm === 'pay-as-you-go'
+                                    ? 'PAYG'
+                                    : displayBudgetName(activePlan, {title: true}),
```

Comment:
can we also keep this sentence case for OD in the whole cart? i think it should only title case if it's at the start of the sentence